src/utils/src/ExponentialBackoffUtils.c

```diff
@@ -0,0 +1,223 @@
+#include "Include_i.h"
+
+//#define LOG_CLASS "XXXXX"
+
+STATUS getDefaultExponentialBackOffConfig(PExponentialBackoffConfig* ppPExponentialBackoffConfig) {
```

Comment:
Consider naming the prefix setupDefault... or initializeDefault instead of get

Reply:
Changed to `initializeDefaultExponentialBackOffConfig`